main.cjs

```diff
@@ -0,0 +1,79 @@
+const { app, BrowserWindow, session, ipcMain, shell } = require("electron");
+const url = require("url");
+const isDev = require("electron-is-dev");
+const path = require("path");
+
+const createWindow = () => {
+  const win = new BrowserWindow({
+    show: false,
```

Comment:
This part makes it so that when the binary is launched it is not displayed, later on in the code we are explicitly showing the UI.